include/triton/Dialect/TritonNvidiaGPU/IR/TritonNvidiaGPUOps.td

```diff
@@ -34,6 +34,7 @@ include "mlir/Interfaces/SideEffectInterfaces.td" // Pure
 include "mlir/Interfaces/InferTypeOpInterface.td" // SameOperandsAndResultType
 include "mlir/Interfaces/DestinationStyleOpInterface.td"
 include "mlir/Interfaces/ViewLikeInterface.td"
+include "triton/Dialect/TritonGPU/IR/TritonGPUTypes.td"
```

Comment:
nit: is this needed? I can't find why :)

Reply:
I swear I did not add this!